gammapy/modeling/parameter.py

```diff
@@ -248,6 +249,14 @@ def min(self, val):
         else:
             self._min = self._set_quantity_str_float(val)
 
+    @property
+    def factor_min(self):
```

Comment:
Per curiosity, for which purposes the two function factor_min and factor_max have been introduced? It seems not directly related to the PR topics